modules/telemetry/v1/data_block.py

```diff
@@ -304,6 +274,97 @@ def __iter__(self):
+        self.sd_checkouts_missed: int = sd_checkouts_missed
+
+    def __len__(self) -> int:
+        return 16
+
+    @classmethod
+    def from_bytes(cls, payload: bytes):
+        parts = struct.unpack("<IIII", payload)
+
+        try:
+            kx134_state = SensorStatus((parts[1] >> 16) & 0x7)
+        except ValueError as error:
+            raise DataBlockException(f"Invalid KX134 state: {(parts[1] >> 16) & 0x7}") from error
+
+        try:
+            alt_state = SensorStatus((parts[1] >> 19) & 0x7)
+        except ValueError as error:
+            raise DataBlockException(f"Invalid altimeter state: {(parts[1] >> 19) & 0x7}") from error
+
+        try:
+            imu_state = SensorStatus((parts[1] >> 22) & 0x7)
+        except ValueError as error:
+            raise DataBlockException(f"Invalid IMU state: {(parts[1] >> 22) & 0x7}") from error
+
+        try:
+            sd_state = SDCardStatus((parts[1] >> 25) & 0x7)
+        except ValueError as error:
+            raise DataBlockException(f"Invalid SD card state: {(parts[1] >> 25) & 0x7}") from error
+
+        try:
+            deployment_state = DeploymentState((parts[1] >> 28) & 0xF)
+        except ValueError as error:
+            raise DataBlockException(f"Invalid deployment state: {(parts[1] >> 28) & 0xf}") from error
+
+        return StatusDataBlock(
+            parts[0], kx134_state, alt_state, imu_state, sd_state, deployment_state, parts[2], parts[3]
+        )
+
+    def to_payload(self) -> bytes:
+        """Transforms a StatusData block into a byte payload."""
+        kx134_state = (self.kx134_state.value & 0x7) << 16
+        alt_state = (self.alt_state.value & 0x7) << 19
+        imu_state = (self.imu_state.value & 0x7) << 22
+        sd_state = (self.sd_state.value & 0x7) << 25
+        deployment_state = (self.deployment_state.value & 0x7) << 28
+
+        states = kx134_state | alt_state | imu_state | sd_state | deployment_state
+
+        return struct.pack("<IIII", self.mission_time, states, self.sd_blocks_recorded, self.sd_checkouts_missed)
+
+    def __str__(self):
+        return (
+            f"{self.__class__.__name__} -> time: {self.mission_time} ms, kx134 state: "
+            f"{str(self.kx134_state)}, altimeter state: {str(self.alt_state)}, "
+            f"IMU state: {str(self.imu_state)}, SD driver state: {str(self.sd_state)}, "
+            f"deployment state: {str(self.deployment_state)}, blocks recorded: "
+            f" {self.sd_blocks_recorded}, checkouts missed: {self.sd_checkouts_missed}"
+        )
+
+    def __iter__(self):
+        yield "mission_time", self.mission_time
+        yield "kx134_state", self.kx134_state
+        yield "altimeter_state", self.alt_state
+        yield "imu_state", self.imu_state
+        yield "sd_driver_state", self.sd_state
+        yield "deployment_state", self.deployment_state
+        yield "blocks_recorded", self.sd_blocks_recorded
+        yield "checkouts_missed", self.sd_checkouts_missed
+
+
```

Comment:
I'm sorry you wrote all this 😭 but there is no status data block in v1. We aren't recording any of this information (altimeter state, blocks recorded, etc.). Could you please remove this entirely?

Reply:
No worries, it was all copy paste.

The reason I re-added `StatusDataBlock` and all of its dependencies was because `telemetry.py` has a field `status` which is of type `StatusData` which requires `RocketData` which requires `StatusDataBlock`. 

I'm seeing that the `RocketData` field isn't being used atm. I can remove both classes in that case in this PR.

Reply:
I would leave RocketData for now, its a byproduct of v1 not being finalized.

Reply:
I would keep the StatusData class, and remove RocketData as a field within.

Reply:
Additionally, yes nuke statusdatablock as its done differently in v1

Reply:
👍 out of curiosity, how is it being done in v1?

Reply:
> 👍 out of curiosity, how is it being done in v1?

Currently, it's not.